src/Lattice/tests/test_bconds.cpp

```diff
@@ -96,13 +96,13 @@ TEST_CASE("periodic_bulk_bconds", "[lattice]")
   REQUIRE(cl->outOfBound(v4) == false);
 }
 
-TEST_CASE("uniform 3D grid layout", "[lattice]")
+TEST_CASE("uniform 3D Lattice layout", "[lattice]")
 {
-  Uniform3DGridLayout grid;
-  grid.BoxBConds = true; // periodic
+  CrystalLattice<OHMMS_PRECISION, OHMMS_DIM> Lattice;
+  Lattice.BoxBConds = true; // periodic
 
-  grid.R.diagonal(1.0);
-  grid.reset();
+  Lattice.R.diagonal(1.0);
+  Lattice.reset();
```

Comment:
There seem to be no requirements for this test.

Reply:
This test seems useless. Removed it.

Reply:
I reverted the removal and added some checks.